qiita_db/reference.py

```diff
@@ -23,6 +79,7 @@ class Reference(QiitaObject):
     sequence_fp
     taxonomy_fp
     tree_fp
+    sortmerna_indexed_db
```

Comment:
True it will raise on that property -- I was thinking about the `create` warning. I think the property should neither raise nor warn, just return None if it doesn't exist.